portfolio/src/main/webapp/index.html

```diff
@@ -21,11 +21,11 @@ <h1>Welcome!</h1>
         <br>
 
         <div class = "subParagraphs">
-        	<div class = "subPar" onclick="expand('about')"><h2 class = "link">About me</h2></div>
-        	<div class = "subPar" onclick="expand('projects')"><h2 class = "link">Past Projects</h2></div>
-            <div class = "subPar" onclick="expand('facts')"><h2 class = "link">Fun Facts</h2></div>
+        	<div class = "subPar" id = "about" onclick="expand('about')"><h2 class = "link">About me</h2></div>
+        	<div class = "subPar" id = "projects" onclick="expand('projects')"><h2 class = "link">Past Projects</h2></div>
+            <div class = "subPar" id = "facts" onclick="expand('facts')"><h2 class = "link">Fun Facts</h2></div>
          </div>
-        <div id= "about-container"></div>
+        <center><div id = "about-container"></div></center>
```

Comment:
I would be careful with the tags you use the center tag was discontinued in HTML5 its better if you style it with text-align:center; or manipulate it with positioning